lib/error.js

```diff
@@ -50,10 +50,12 @@ function error(source, position, current, message, type) {
   const contextualMessage = precedingLine + sobsequentLine + "\n" + spaced;
 
   const contextType = type === "Syntax" ? "since" : "inside";
+  const fileContext = source.fileName ? ` in ${source.fileName}` : "";
```

Comment:
Super annoying nit: filename is just one word in English 🥺

Reply:
nit 2: it's not always for a file. What would be a better name? `trackingName`? Just `name`?

Reply:
`name` could work :)

Reply:
I'll delete my suggestions.